tests/test_mediapackage/test_mediapackage.py

```diff
@@ -81,6 +84,16 @@ def test_describe_channel_succeeds():
     describe_response["Tags"]["Customer"].should.equal("moto")
 
 
+@mock_mediapackage
+def test_describe_unknown_channel_throws_error():
+    client = boto3.client("mediapackage", region_name=region)
+    channel_id = "unknown-channel"
+    with pytest.raises(ClientError) as err:
+        client.describe_channel(Id=channel_id)
+    assert err.value.response["Error"]["Code"] == "NotFoundException"
+    assert err.value.response["Error"]["Message"] == f"'{channel_id}'"
```

Comment:
Python2 doesn't support f-strings, looks like that's causing the build to fail. Happy to merge once the CI is happy though 👍

Reply:
I fixed it. But although I ran black, version 19.10b0 or even make format, make lint, and no changes were done. But still, CI is failing. Any hint? Sorry for the hassle.

Reply:
@arcovoltaico Python 2 created a unicode-string out of the Exception that was thrown. When running the test, it was comparing a normal string with an unicode-string - and that was failing. 
```
assert "u'unknown-channel'" == "'unknown-channel'"
```
Running this scenario against AWS showed that the error message should be slightly different anyway, so I reworked the logic to return the correct error message